airbyte-api/build.gradle

```diff
@@ -99,18 +99,6 @@ task generateApiDocs(type: GenerateTask) {
             generatePom  : "false",
             interfaceOnly: "true"
     ]
-
-    doLast {
```

Comment:
Why is this removed instead of just changing the target?

Reply:
I just changed the path instead. Now it should still auto-update docs instead of keeping a stale copy.